sos/report/plugins/openstack_nova.py

```diff
@@ -214,6 +215,7 @@ class RedHatNova(OpenStackNova, RedHatPlugin):
     apachepkg = "httpd"
     nova = False
     packages = ('openstack-selinux',)
+    postproc_dirs = ["/etc/nova/", "/var/lib/openstack/config/nova"]
```

Comment:
what about just adding to the list, if the common directories increase, then you don't have to change it here? I know it's unlikely, but something to think about

Reply:
You mean always iterate in `apply_regex_sub` method over:

    for _dir in ["/etc/nova/", "/var/lib/openstack/config/nova"]:

? Otherwise I dont understand.